### PR TITLE
fix: network_bandwidth for mac

### LINKED ISSUE #1: [BUG]: Not getting network bandwidth in my mac m1 air
### Description of the bug

image
I used your plugin but not getting network-bandwidth

### Steps To Reproduce

1. Just add the plugin to `.tmux.conf`
2. launch the tmux

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by shubhamku044**
## What

What does this pull request accomplish?

- [x] Finished item
- [ ] Item in progress
- [x] Fixes #1

## How

What code changes were made to accomplish it?

in `network_bandwidth.sh` file, I made a check whether `uname -s` is `Darwin` or `Linux`. then I added extra lines inside a if else block for `Darwin`

## Why

- [ ] I want to fix the issue # (issue)
- [x] I want to add a new feature
- [ ] I want to improve the documentation
- [ ] I want to fix the tests

## Screenshots (If applicable)

image

## Notes

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: scripts/network_bandwidth.sh [hunk 1]
```diff
@@ -2,7 +2,18 @@
 
 INTERVAL="1"  # update interval in seconds
 
-network_name=$(tmux show-option -gqv "@tmux2k-network-bandwidth")
+# Network interface to monitor
+network_name="en0"
```
Comment (2KAbhishek) [inline scripts/network_bandwidth.sh:6]: Hardcoding the network name doesn't feel like a good idea, we can use network name passed in as an option for better usability

**Comment by 2KAbhishek** [comment]
fixes #2 thanks @shubhamku044

File: scripts/network_bandwidth.sh (hunk 1 above)
Comment (shubhamku044) [reply scripts/network_bandwidth.sh:6]: so I thought that in mac or any laptop, people mostly use wifi to connect to the internet, and this `en0` is the wifi. But, I will change it.

Comment (2KAbhishek) [reply scripts/network_bandwidth.sh:6]: I'm merging this in, will add the name option and docs

Comment (shubhamku044) [reply scripts/network_bandwidth.sh:6]: Okay

**Pull request merged**